modules/storages/spec/lib/api/v3/file_links/file_link_representer_parsing_spec.rb

```diff
@@ -64,24 +77,55 @@
       end
 
       it 'are not used by the parsing' do
-        expect(parsed).not_to have_attributes(
-          created_at: DateTime.parse(parsed_hash["createdAt"]).in_time_zone,
-          updated_at: DateTime.parse(parsed_hash["updatedAt"]).in_time_zone
-        )
+        expect(parsed).not_to have_attributes(created_at: DateTime.parse(parsed_hash["createdAt"]).in_time_zone,
```

Comment:
About this change in style, is there a particular reason to choose it? 
I personally prefer having 
```
method(
  param1,
  param2
)
```
instead of 
```
method(param1,
       param2)
```
because it keeps the git diff shorter and relevant when the method name changes:
```
new_method_name(
  param1,
  param2
)
```
Also it gives shorter line lengths and deals better with indentation.

As this style was not prohibited by a rubocop rule, I thought it was okay to use it. Should I stick to using the "FirstMethodParameterLineBreak" style everywhere?